components/automate-chef-io/content/docs/desktop.md

```diff
@@ -0,0 +1,76 @@
+Selecting rows within these displays will list relevant desktops and selecting an singular desktop reveals its detailed information.
+
+The _Desktop_ dashboard does not support project filtering.
+Node counts in the _Desktop_ dashboard may include liveness agents.
+
+### Daily Check-in
+
+The _Daily Check-in_ display shows a top-level view of daily desktop check-in statistics.  
+A bar graphic illustrates the proportion of desktops with `unknown` and `checked-in` statuses.
+Below the bar, boxes displays counts for desktops with `unknown` and `checked-in` statuses, and a total count of desktops.
+`Checked-in` refers to desktops reporting into Chef Automate.
+`Unknown` desktops did not report to Chef Automate during the last twenty-four hours.
+
+### Check-in History
+
+_Check-in History_ shows the history of checked-in desktop counts as graphed over a selected period of time.
+Adjust the timescale with the timescale drop-down menu and change the graph display.
+The timescale options are "Last 3 Days," "Last 7 Days," and "Last 14 Days".
+
+### Top 10 Errors
+
+The _Top 10 Errors_ display shows the ten most common errors that have occurred across all desktops over the last twenty-four hours, and a count of the machines that experienced each error.
+Selecting an error opens the Filter Desktop List with the applied error filter to enable further investigation.
+
+### Time Since Last Check-in
+
+The _Time since Last Check-in_ display shows a count of desktops with an `unknown` status for defined timefames.
+Selecting a row will reveal a Filtered Desktop List of desktops for the selected timeframe.
+
+### Filtered Desktop Lists
+
+The Filtered Desktop List displays a list of desktops filtered according to the selected timeframe or error in the previous display.
+Selecting rows in either _Time since Last Check-in_ or _Top 10 Errors_ activates this summary list.
+
+Apply more filters to this initial list if desired.
+Specific filter values populate according to available desktop information and include the categories of "platform," "version," "domain," and "last run status."
+
+Select an individual desktop row to display the individual desktop's node details.
+
+#### Node Details
+
+The _Node Details_ display shows the details of a single desktop and includes a visualization of its check-in history.
+
+The individual check-in history displays the desktop's latest status for each day with an associated icon by default.
+Statuses include "converged" (check box icon), "unchanged" (dash icon), "error" (exclamation point icon), or "unknown" (question mark icon).
+The check-in history's format can appear in grid icon, status list, or selectable JSON form.
+Select the gear icon at the top of the display to make these modifications.
+The timescale of the check-in history can also switch between "Last 2 Weeks" and "Last 4 Weeks."
+
+The individual desktop's information includes a detailed overview, Chef Infra Client, System, Virtualization, and Kernel.
```

Comment:
```suggestion
The individual desktop has an information panel that displays the following data: Overview, Chef Infra Client, System, Virtualization, and Kernel.
```

Reply:
THANK YOU! I was struggling with how to phrase that info.